test/Microsoft.AspNet.Diagnostics.Tests/TestFileProvider.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.Framework.Caching;
+
+namespace Microsoft.AspNet.Diagnostics
+{
+    public class TestFileProvider : IFileProvider
```

Comment:
Move to `private class` in `ErrorPageMiddlewareTest`, it's not used anywhere else. Ditto with `TestFileInfo.cs`